extensions/interactions/RatioExpressionInput/directives/ratio-expression-input-validation.service.spec.ts

```diff
@@ -297,6 +297,17 @@ describe('RatioExpressionInputValidationService', () => {
     }]);
   });
 
+  it('should catch integral value greater than 10 for # terms', () => {
+    customizationArgs.numberOfTerms.value = 11;
+    warnings = validatorService.getAllWarnings(
+      currentState, customizationArgs, answerGroups,
+      goodDefaultOutcome);
+    expect(warnings).toEqual([{
+      type: WARNING_TYPES.ERROR,
+      message: ('The number of terms in a ratio should not be more than 10.')
```

Comment:
*nit - I don't think you need to add parens if the string fits in a single line.